source/clear-linux/tutorials/kubernetes.rst

```diff
@@ -338,13 +338,13 @@ Troubleshooting
 
         sudo source profile
 
-  #. To ssure your system isn't running previous session variables, run:
+  #. To assure your system isn't running previous session variables, run:
 
      .. code-block:: bash
 
         sudo kubeadm reset --cri-socket=/run/crio/crio.sock
 
-  #. Return to the `kubeadm init` command above.
+  #. Continue below and pass the `-E` in the command.
```

Comment:
Is this -E option really necessary?  I find that if when I complete the control plane initialization and then run these commands which is what kubeadm says you should run after it is done I no longer have to use -E to run kubectl and kubeadm reset. It doesn't hurt to use -E though. 

mkdir -p $HOME/.kube
sudo cp  /etc/kubernetes/admin.conf $HOME/.kube/config
sudo chown $(id -u):$(id -g) $HOME/.kube/config

Reply:
The tutorial suggests using -E only at the control plane initialization (kubeadm init). You do not need it afterwards.

Reply:
I don't have to use the -E option though even at init as long as I copy the above but I also did not test using a proxy.  It doesn't hurt to have it there.

Reply:
Context, please. This addition is in the `Troubleshooting` section. So it's catering to those who may have proxy issues.  The original step, above, remains without `-E`.

Reply:
Leave it in then. I am good with that.